src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/ServiceFabricClusterCertificateCmdlet.cs

```diff
@@ -127,9 +127,28 @@ public virtual string CertificateSubjectName
                 {
                     this.certificateSubjectName = string.Concat("cn=", value);
                 }
+                else
+                {
+                    this.certificateSubjectName = value;
+                }
             }
         }
 
+        [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = ByExistingKeyVault,
```

Comment:
Remove valuefrompipeline from these two parameters as well. It doesn't make sense for a string parameter to have this attribute.

Reply:
done, thanks